docusaurus/docs-internals/editor-controls/multivalue-controls/transform.md

```diff
@@ -12,15 +12,35 @@ Example of the `transform`:
+| `states?`          | `Array<State>`                                                                                                                                                                             | [`"normal"`] | Allows for different styles based on the element's state <br/> <br/> <b>`State = "normal" \| "hover" \| "active"`</b> <br/> <br/> `"normal"` - the normal state of an element, <br/> `"hover"` - the state when the element is hovered over, <br/> `"active"` - the state when the element is active (e.g., current page in pagination)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
i think we need to remove `states` because it look so bad

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/f640f03e-3fd4-48a2-b0cf-5d91b1ec19c2)


also don't forget to remove the states examples